lib/include/openamp/virtqueue.h

```diff
@@ -88,7 +84,7 @@ struct virtqueue {
 	uint16_t vq_available_idx;
 
 #ifdef VQUEUE_DEBUG
-	boolean vq_inuse;
+	bool vq_inuse;
```

Comment:
the OpenAMP defines boolean because some compiler doesn't have bool defined. Better to define bool in generic compiler header if it is not defined.

Reply:
but stdbool.h is already included by rpmsg.h and virtqueue.h, according this link:
http://pubs.opengroup.org/onlinepubs/009695399/basedefs/stdbool.h.html
the purpose of stdbool.h is just for bool/false/true definition, so we could safely assume bool is defined if stdbool.h exist.

Reply:
Good catch. just bool doesn't exist in C89/90. and thus, for those C compilers, there will be issue.
but, as you point out, there is inconsistency in the current code base. I am fine to just use bool and remove the boolean. Please also remove "typedef uint8_t boolean;" in lib/include/openamp/virtqueue.h

Reply:
Already remove in the same patch